crates/typst/src/layout/frame.rs

```diff
@@ -533,6 +533,40 @@ pub enum FrameItem {
     Tag(Tag),
 }
 
+impl FrameItem {
+    pub(in crate::layout) fn size(&self) -> Option<Size> {
```

Comment:
I'd prefer to have a way of accessing the size without having to `compose` each child individually. I hope this is fine. It generally _should work_.